files/uk/web/html/element/input/submit/index.md

```diff
@@ -0,0 +1,223 @@
+---
+title: <input type="submit">
+slug: Web/HTML/Element/input/submit
+page-type: html-element
+browser-compat: html.elements.input.type_submit
+---
+
+{{HTMLSidebar}}
+
+Елементи {{HTMLElement("input")}} типу **`submit`** (подати) візуалізуються як кнопки. Коли трапляється подія {{domxref("Element/click_event", "click")}} (зазвичай у зв'язку з тим, що користувач клацнув кнопку), то {{Glossary("User agent", "користувацький агент")}} пробує подати форму на сервер.
+
+## Значення
+
+Атрибут [`value`](/uk/docs/Web/HTML/Element/input#value) елемента `<input type="submit">` містить рядок, що виводиться як підпис кнопки.
```

Comment:
```suggestion
Атрибут [`value`](/uk/docs/Web/HTML/Element/input#value) елемента `<input type="submit">` містить рядок, що виводиться як підпис кнопки. Інакше кнопка не міститиме справжнього значення.
```

Не вистачає перекладу "Buttons do not have a true value otherwise". Пропоную такий варіант)

Reply:
```suggestion
Атрибут [`value`](/uk/docs/Web/HTML/Element/input#value) елемента `<input type="submit">` містить рядок, що виводиться як підпис кнопки. Без нього кнопка не матиме фактичного значення.
```
Пропоную так

Reply:
> Пропоную так

Супер, давай цей варіант.